optaplanner-core/src/test/java/org/optaplanner/core/impl/solver/DefaultSolverTest.java

```diff
@@ -116,7 +116,7 @@ public void solveStopsWhenPartiallyInitialized() {
     }
 
     @Test(timeout = 600_000)
-    public void zeroEntity() {
+    public void solveThrowsWhenZeroEntity() {
```

Comment:
How about `solveThrowsExceptionWhenZeroEntity`? I find it easier to read as a sentence.